zipkin-lens/package-lock.json

```diff
@@ -8467,9 +8467,9 @@
       "integrity": "sha512-N+8UisAXDGk8PFXP4HAzVR9nbfmVJ3zYLAWiTIoqC5v5isinhr+r5uaO8+7r3BMfuNIufIsA7RdpVgacC2cSpw=="
     },
     "node_modules/three": {
-      "version": "0.106.2",
-      "resolved": "https://registry.npmjs.org/three/-/three-0.106.2.tgz",
-      "integrity": "sha512-4Tlx43uoxnIaZFW2Bzkd1rXsatvVHEWAZJy8LuE+s6Q8c66ogNnhfq1bHiBKPAnXP230LD11H/ScIZc2LZMviA=="
+      "version": "0.161.0",
```

Comment:
manually fixed the CVE again, as the last commit reverted the exception. That's completely unsurprising. Anyway, good news is no critical CVEs left (according to trivy)